pkg/store/container/container.go

```diff
@@ -89,6 +91,16 @@ func (c *Container) Delete() error {
 	return c.Status.Delete()
 }
 
+// Stop close stopCh of the container.
+func (c *Container) Stop() {
```

Comment:
nit: Instead of adding a function to `Container`, can we make a type of `stopCh` and add function to it instead?

In this way, it can be reused in Sandbox after my change is in.

Reply:
Will do.

Reply:
Done.